sdk-scheme-adapter/outbound/v2_0_0/components/schemas/individualTransferFulfilment.yaml

```diff
@@ -0,0 +1,13 @@
+type: object
```

Comment:
Same comment applies here as this looks to be part of the standard FSPIOP Bulk definition.